testing/e2e/skr-tester/pkg/keb/client.go

```diff
@@ -0,0 +1,341 @@
+package keb
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+const (
+	SCOPES                     = "broker:write"
+	KYMA_SERVICE_ID            = "47c9dcbf-ff30-448e-ab36-d3bad66ba281"
+	trialPlanID                = "7d55d31d-35ae-4438-bf13-6ffdfa107d9f"
+	DEFAULT_EXPIRATION_SECONDS = 600
```

Comment:
inconsistent naming convention: constants with caps or snakeCase?

Reply:
Changed to snakeCase.